lib/api/client-commands/a11y/runTests.js

```diff
@@ -0,0 +1,54 @@
+const ClientCommand = require('../_base-command.js');
+
+/**
+ * Analyzes the current page against applied axe rules.
+ *
+ * @example
+ * describe('accessibility testing', function() {
+ *   it('accessibility rule subset', function(browser) {
+ *     browser
+ *       .url('https://www.w3.org/WAI/demos/bad/after/home.html')
+ *       .assert.titleEquals('Welcome to CityLights! [Accessible Home Page]')
+ *       .a11y.runTests('body', {
+ *         runOnly: ['color-contrast', 'image-alt'],
+ *       });
+ *   });
+ * })
+ *
+ * @syntax .a11y.runTests(selector, [callback])
+ * @syntax .a11y.runTests(selector, options, [callback])
+ * @method a11y.runTests
+ * @param {string} selector The CSS selector used to locate the element.
+ * @param {object} [options] Object containing [rules configuration](https://www.deque.com/axe/core-documentation/api-documentation/#user-content-options-parameter) to use when performing the analysis.
+ * @param {function} callback Callback function which is called with the result value.
+ * @link https://github.com/dequelabs/axe-core
+ * @api protocol.accessibility
+ * @since 2.3.6
+ */
+class AxeRunTests extends ClientCommand {
+  static get isTraceable() {
+    return true;
+  }
+
+  performAction(callback) {
+    this.api.axeInject().axeRun(this.axeSelector, this.axeOptions, callback);
```

Comment:
See https://github.com/nightwatchjs/nightwatch/pull/3610/files#r1127848507

Reply:
I'm not sure how to do this because axe commands are overridden by the plugin and hence these commands are not directly available in transport mappings.